gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/src/DbDataStoreUtils.js

```diff
@@ -124,7 +112,6 @@ gpii.oauth2.dbDataStore.cleanUpDoc = function (data) {
         data.id = data._id;
         delete data._id;
         delete data._rev;
-        delete data.type;
```

Comment:
The function comment documentation still includes "type" though it is no longer deleted.